Validation/HcalDigis/interface/HcalDigisClient.h

```diff
@@ -78,6 +78,21 @@ class HcalDigisClient : public DQMEDHarvester {
         msm_->find(name)->second->Fill(X, weight);
     }
 
+    //TP Code
+    void projectY(std::string name_to, TH2F* TH2F_from){
```

Comment:
isn't this exactly what root does in the profiling method?
see for example: https://root.cern.ch/root/html/TH2.html#TH2:ProfileX